pkg/gui/controllers/custom_command_action.go

```diff
@@ -20,7 +20,7 @@ func (self *CustomCommandAction) Call() error {
 		HandleConfirm: func(command string) error {
 			if self.shouldSaveCommand(command) {
 				self.c.GetAppState().CustomCommandsHistory = utils.Limit(
-					lo.Uniq(append(self.c.GetAppState().CustomCommandsHistory, command)),
+					lo.Uniq(append([]string{command}, self.c.GetAppState().CustomCommandsHistory...)),
```

Comment:
This is not the most efficient way to create the new slice, but it's the shortest, and I think performance doesn't matter too much in this case.